miso-web/src/main/webapp/pages/editProject.jsp

```diff
@@ -139,7 +139,20 @@
         </form:select>
     </td>
   </tr>
-  <c:if test="${project.id != 0 && !projectReportLinks.isEmpty()}">
+  <c:if test="${project.id != 0}">
+  <c:if test="${detailedSample}">
+  <tr>
+    <td>Subprojects:</td>
+    <td>
+    <c:forEach items="${subprojects}" var="subproject">
+      <button class="ui-state-default ui-state-hover small-gap-right clickable-non-link"
+        onclick="Subproject.filterSamples('samples_section_arrowclick', 'project_samples', '${subproject.alias}'); return false;">${subproject.alias}</button>
+    </c:forEach>
+    <br/><a href="<c:url value='/miso/subproject/bulk/new?quantity=1'/>">Add new subproject</a>
```

Comment:
This should pre-populate the project field in the subprojects table...

Reply:
Would be ideal, but would require heavy refactoring as that path is handled by the AbstractInstituteDefaultsController... could I get your opinion on the best way to approach that?